responses.go

```diff
@@ -338,19 +339,26 @@ func (sir SyncInvitedRoom) MarshalJSON() ([]byte, error) {
 	return util.MarshalAndDeleteEmpty((marshalableSyncInvitedRoom)(sir), syncInvitedRoomPathsToDelete)
 }
 
+type SyncKnockedRoom struct {
+	State SyncEventsList `json:"knock_state"`
+}
+
 type RespTurnServer struct {
 	Username string   `json:"username"`
 	Password string   `json:"password"`
 	TTL      int      `json:"ttl"`
 	URIs     []string `json:"uris"`
 }
 
-type RespAliasCreate struct{}
-type RespAliasDelete struct{}
-type RespAliasResolve struct {
-	RoomID  id.RoomID `json:"room_id"`
-	Servers []string  `json:"servers"`
-}
+type (
+	RespAliasCreate  struct{}
+	RespAliasDelete  struct{}
+	RespAliasResolve struct {
+		RoomID  id.RoomID `json:"room_id"`
+		Servers []string  `json:"servers"`
+	}
+)
```

Comment:
does this formatting change actually do anything besides group these together?

If we are grouping `RespAlias*` together, why is `RespAliasList` not included here?

Reply:
Woa! Not sure how this slipped in think this was `gofumpt` sneaking this in here let me fix that.